eng/test-mounts.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -xeuo pipefail
+IFS=$'\n\t'
+
+# We take the snap name as an input parameter
+snap="$1"
+dotnet_path="$2"
```

Comment:
question(non-blocking): Why is `dotnet_path` passed as a parameter instead of assembled in the script like `content_snap_path`?

Reply:
Long story short: the `dotnet-runtime-*` snaps historically place the .NET directory hive in `/`, whereas all the other ones are in `/usr/lib/dotnet`. So, in the script invocations within each content snap workflow, I need to pass the .NET root as a parameter.

In any case, this is a temporary variable. Once I have the runtime content snaps using our package, the .NET directory hive should go to `/usr/lib/dotnet` like the others and this will no longer be necessary.